<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1144</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>北交视觉实践-视频增强</string>
  </property>
  <widget class="QLabel" name="lblOriginal">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>501</width>
     <height>441</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpenCamera">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>570</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;幼圆&quot;;
font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>打开摄像头</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;隶书&quot;;
font: 28pt &quot;隶书&quot;;</string>
   </property>
   <property name="text">
    <string>视频感知增强系统</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPauseOrResume">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>580</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Ritinex">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>570</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;</string>
   </property>
   <property name="text">
    <string>Ritinex增强</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblProcessed_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>60</y>
     <width>501</width>
     <height>441</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>530</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>增强前</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>520</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;幼圆&quot;;</string>
   </property>
   <property name="text">
    <string>增强后</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>10</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Version1.0 By 北交朱宏梁</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
